javascript/index.js

```diff
@@ -14,52 +14,85 @@ const milUniElement = document.getElementById('milUni');
 const splitsElement = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+printMinutes() 
+printSeconds();
 }
 
 function printMinutes() {
-  // ... your code goes here
+ const minutes = chronometer.getMinutes();
+ minUniElement.innerText = chronometer.computeTwoDigitNumber(minutes)[1];
+ minDecElement.innerText = chronometer.computeTwoDigitNumber(minutes)[0];
+
 }
 
 function printSeconds() {
-  // ... your code goes here
+  const seconds = chronometer.getSeconds();
+  secUniElement.innerText = chronometer.computeTwoDigitNumber(seconds)[1];
+  secDecElement.innerText = chronometer.computeTwoDigitNumber(seconds)[0];
 }
 
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+  
 }
 
 function printSplit() {
-  // ... your code goes here
+const newElm = document.createElement('li');
+
+
 }
 
 function clearSplits() {
-  // ... your code goes here
+  
 }
 
 function setStopBtn() {
-  // ... your code goes here
+  btnLeftElement.className = 'btn start';
+  btnLeftElement.innerText = 'START';
 }
 
```

Comment:
normally it would make more sense that you do stop like this:
```
 btnLeftElement.className = 'btn stop';
 btnLeftElement.innerHTML = 'STOP';
```
same for the other functions, but in the end it only matters how and when you invoke the functions